### PR TITLE
Adds kitchen knife to cytology closet prefab

### LINKED ISSUE #84466: The icebox cytology equipment locker lacks a kitchen knife.
Reporting client version: 515.1640

## Issue Summary

As in the title. The icebox cytology locker should have a kitchen knife like Metastation does so that way cytologists can cut up monkeys for protein, since protein is used in like 98% of cytology recipes.

## Round ID:
[231341](https://scrubby.melonmesa.com/round/231341)

## Testmerges:
- [[TM ONLY] Hijacked Revision Date for v3 TGS Api](https://www.github.com/tgstation/tgstation/pull/83227)
- [Some alt & ctrl click improvements](https://www.github.com/tgstation/tgstation/pull/84203)
- [Moves tool use back higher in the chain, but makes it so tool acts are only called on non-combat-mode](https://www.github.com/tgstation/tgstation/pull/84083)

## Reproduction:

Open the locker in xenobiology.

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request

A kitchen knife is very useful to people doing cytology, since a lot of samples come from dead monkeys. Meta and Wawa were the only maps which actually started with one, so I removed those and just added it to the prefab locker that spawns in every Cytology Lab.
## Why It's Good For The Game

Fixes #84466.

Important tools should be available to everyone doing the job that requires them.
## Changelog
:cl: Vekter
fix: Standardized the availability of a kitchen knife for cytology on all maps.
/:cl:

### CODE REVIEW

**Comment by EuSouAFazer** [comment]
Bad PR, with various issues.

1: If this is to go through you'd also have to remove the kitchen knife from Tram and the hunting knife from Northstar.
You removed from Meta and Wawa but Tram and Northstar also already have ways of butchering monkeys. Only Delta, Birdshot and Icebox don't have a way of doing it. If this is to be merged then these knifes need to be removed as to not double them up on knifes.

2: Unecessary as of https://github.com/tgstation/tgstation/pull/84235 
Cytologists start with 40 protein, which is enough for the entire round for most rounds. Butchering monkeys isn't a mandatory part of the job anymore; cytologists can easily go the round without butchering a monkey now.

3: Conceptually against the mapping of Cytology. 
The way Cytology is setup right now every map has ups and downs compared to others. For example, Northstar starts with a pig and a cow, Tram starts with two patches of soil rather than one, Meta starts with Tofu for free protein, etc. In fact, **Birdshot is by far the best map to do Cytology on right now**, due to starting with neglected fern, moisture trap and a moldy pizza. It doesn't make sense to further buff it by giving a knife.

Making the knife a part of the locker means we take away mapper freedoms from having unique cytology setups; Northstar's hunting knife & laser gun wouldn't be possible, and we'd need to take away something else from Birdshot to balance the extra addition there. Standarization is not cool here, the idea is every lab has pros and cons.

But also, 
4: This doesn't solve the actual issues of Icebox.
Icebox has a _horrible_ cytology map. Icebox's cytology map **does not have a growing vat**. Which is fucking absurd now that the RCD is no longer in their locker. We literally don't have the most important machine for it, the thing _actually_ required to do the job. It's impossible to do cytology on Icebox. This is the cytology equivalent of not having a SM crystal or genetics console.

There is also no cytology lab in Icebox, having its equipment literally scattered around the Xenobio map. The microscope is 10 tiles away from the cytology locker, and the smart petri dish storage is 19 tiles away from the locker, wtf? And between all the cytology stuff is all the Xenobio stuff. And of course there's not a unique cytology advantage to Icebox like all other maps have.

Delta also needs to have a unique advantage but it's not as urgent as Icebox literally not having a vat.

------

Basically:
1: This doesn't finish the job, some labs would have 2 knifes
2: Knifes aren't a required item for the job anymore
3: It's against the design idea of the cytology maps; they're meant to be nonstandarized
4: It doesn't fix Icebox. Cytology needs a roundstart growing vat.

**Comment by TheVekter** [comment]
> Bad PR, with various issues.

No u.

> 1: If this is to go through you'd also have to remove the kitchen knife from Tram and the hunting knife from Northstar. You removed from Meta and Wawa but Tram and Northstar also already have ways of butchering monkeys. Only Delta, Birdshot and Icebox don't have a way of doing it. If this is to be merged then these knifes need to be removed as to not double them up on knifes.
I didn't see a kitchen knife in Science for Tram and I'll have to look for that hunting knife on Northstar but I didn't see it anywhere in science on my first pass.

> 2: Unecessary as of #84235 Cytologists start with 40 protein, which is enough for the entire round for most rounds. Butchering monkeys isn't a mandatory part of the job anymore; cytologists can easily go the round without butchering a monkey now.

I don't really care; that's helpful but it's still nice for them to have in case they end up needing more. This isn't really a negative to my PR; you haven't actually stated how them having access to this is bad, just that it might not be necessary.
> 3: Conceptually against the mapping of Cytology. The way Cytology is setup right now every map has ups and downs compared to others. For example, Northstar starts with a pig and a cow, Tram starts with two patches of soil rather than one, Meta starts with Tofu for free protein, etc. In fact, **Birdshot is by far the best map to do Cytology on right now**, due to starting with neglected fern, moisture trap and a moldy pizza. It doesn't make sense to further buff it by giving a knife.

This is bad design; jobs should relatively be the same from map to map and we've historically tried to keep feature parity on most maps. If a maptainer argues that this is somehow critical to how the maps are designed, I'll close this, but you are not a maptainer, so I'm going with what we've done in the past.

> Making the knife a part of the locker means we take away mapper freedoms from having unique cytology setups; Northstar's hunting knife & laser gun wouldn't be possible, and we'd need to take away something else from Birdshot to balance the extra addition there. Standarization is not cool here, the idea is every lab has pros and cons.

I think you make a good point as far as the hunting knife and laser gun on Northstar, so I would be willing to return that to that map but would also just map the knife back into the other cytology labs.
 
> But also, 4: This doesn't solve the actual issues of Icebox. Icebox has a _horrible_ cytology map. Icebox's cytology map **does not have a growing vat**. Which is fucking absurd now that the RCD is no longer in their locker. We literally don't have the most important machine for it, the thing _actually_ required to do the job. It's impossible to do cytology on Icebox. This is the cytology equivalent of not having a SM crystal or genetics console.

Out of scope, though I'll definitely consider doing a remap of it after this PR. Keep in mind that I'm going to bring it to parity with other maps though.

(On another note, I'm not really sure you get license to complain about one of the cytology labs being worse than the others when you literally just said "it's intentional that some of them are better than others", but go off I guess)

**Comment by TheVekter** [comment]
On another note, can I please make one PR that doesn't piss someone off

**Comment by TheVekter** [comment]
84697 is an alternative I'm okay with.
